src/committee/node.rs

```diff
@@ -233,7 +234,7 @@ pub async fn run_server(
     let ctx = NodeState {
         key_package,
         pubkey_package,
-        signing_tasks: RwLock::new(HashMap::new()),
+        signing_tasks: RwLock::new(CappedHashMap::new(100)),
```

Comment:
can you have `100` be a constant defined in `constants.rs`? Perhaps 1000 is a better number also, I think SigningTasks are relatively small